gradle.properties

```diff
@@ -31,4 +31,6 @@ android.useAndroidX=true
 android.enableD8=true
 android.enableD8.desugaring=true
 android.enableJetifier=true
-android.jetifier.blacklist=shadows
\ No newline at end of file
+android.jetifier.blacklist=shadows
+sonatypeUsername=opensrp
+sonatypePassword=b6Ap_sWUpr+wUqa6@br_sUDrebRa=r7W
```

Comment:
Please move this to `local.properties` and clean these credentials from git history